nucypher/blockchain/eth/agents.py

```diff
@@ -450,6 +450,24 @@ def get_restake_unlock_period(self, staker_address: str) -> int:
         restake_unlock_period = int(staker_info[4])  # TODO: #1348 Use constant or enum
         return restake_unlock_period
 
+    @validate_checksum_address
+    def is_winding_down(self, staker_address: str) -> bool:
+        staker_info = self.get_staker_info(staker_address)
```

Comment:
Just a thought for future PRs: we could make `get_staker_info` return a namedtuple